CONTRIBUTING.md

```diff
@@ -124,31 +124,31 @@ pull requests (PRs).
 To create a new PR, fork the project in GitHub and clone the upstream repo:
 
 ```sh
-$ git clone https://github.com/open-telemetry/opentelemetry-python-contrib.git
-$ cd opentelemetry-python-contrib
+git clone https://github.com/open-telemetry/opentelemetry-python-contrib.git
+cd opentelemetry-python-contrib
 ```
 
 Add your fork as an origin:
 
 ```sh
-$ git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-python-contrib.git
+git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-python-contrib.git
 ```
 
 Run tests:
 
 ```sh
 # make sure you have all supported versions of Python installed
-$ pip install tox  # only first time.
-$ tox  # execute in the root of the repository
+pip install tox  # only first time.
```

Comment:
Should we remove the comments so the code block works for copy/paste without problems?

Reply:
Done